### PR TITLE
purge relayer registry file

### PR DESCRIPTION
**Pull request by kziemianek**
As topic

### CODE REVIEW

File: bitacross-worker/service/src/setup.rs [hunk 1]
```diff
@@ -17,9 +17,10 @@
 */
 
 use crate::error::{Error, ServiceResult};
+use frame_support::storage::child::root;
```
Comment (Kailai-Wang) [inline bitacross-worker/service/src/setup.rs]: Is it redundant?

**Review by Kailai-Wang — APPROVED** [review]
Thanks!

**Review by felixfaisal — APPROVED** [review]
LGTM

File: bitacross-worker/service/src/setup.rs (hunk 1 above)
Comment (kziemianek) [reply bitacross-worker/service/src/setup.rs]: Yes, it was added by IDE autoimport ;)

**Comment by Kailai-Wang** [comment]
We have to resolve the conflict and we are good to go

**Pull request merged**